be/src/formats/orc/apache-orc/c++/src/sargs/PredicateLeaf.cc

```diff
@@ -93,6 +93,17 @@ PredicateLeaf::PredicateLeaf(Operator op, PredicateDataType type, std::string co
     validate();
 }
 
+PredicateLeaf::PredicateLeaf(Operator op, PredicateDataType type, uint64_t columnId,
```

Comment:
what the meaning of the Operator op? is it a pipelline::Operator? if so, use OperatorPtr instead of Operator.

Reply:
That's orc sdk's Operator, not for pipeline::Operator